src/docs/error-handling.dox

```diff
@@ -36,8 +36,8 @@ an object is not available for exclusive access. For example, the
 WT_SESSION::drop or WT_SESSION::verify methods will fail if the object
 has open cursors. Note that internal WiredTiger threads may temporarily
 open cursors on objects (for example, threads performing operations like
-handle sweep or statistics logging), and operations may temporarily fail
-and return \c EBUSY when there are no application cursors open on the object.
+statistics logging), and operations may temporarily fail and return \c EBUSY
+when there are no application cursors open on the object.
```

Comment:
I've been back over WT-3362 and I think @keithbostic is correct that internal threads holding `dhandle->rwlock` in any mode can now cause `WT_SESSION::drop` and `WT_SESSION::verify` to return `EBUSY`.  I've opened WT-3520 to discuss making it configurable whether they wait or not.